src/lib/URLState/types.ts

```diff
@@ -4,6 +4,6 @@ export type CovidGraphDefinition = {
   data_type: string;
   x_axis_bounds: number[];
   y_axis_bounds: number[];
-  animate: boolean;
+  animate: boolean; // Bug: if animate=false, it's treated as true. Must omit in order to use false
```

Comment:
Probably qs is parsing it as a string and I just need to change some setting? Not sure. TODO... :)

Reply:
Oh man, this bug is pretty bad actually. If you edit the graph, then `deserialize` saves it as `animate=false` and then that re-enables animation haha

Reply:
Temporarily ignoring `.animate` due to this bug

https://github.com/prendradjaja/covid-dashboard/pull/13/commits/44c03d6de72a3fdec4aa5c0160f6e637f901ef17

Reply:
QS doesn't support boolean conversion, so your thought here is right -- you'd need to add logic to serialize catching the animate value specifically and setting it to false or true depending on its value

Reply:
Gotcha. Just googled and I see that we can provide `qs` with a custom encoder/decoder if we want to use booleans. For a future PR! :D